tests/test_helper.py

```diff
@@ -0,0 +1,105 @@
+import pandas as pd
+import pytest
+from pydisagg.ihme.age_var import rename_dict_dis
+from pydisagg.ihme.helper import rename_df, glue_back
+
+
+def test_rename_df():
+    # Prepare a test dataframe
+    df = pd.DataFrame(
```

Comment:
This looks good! Thanks @saalUW. One minor comment, if we want to use this data frame repetitively, it might be good to create it outside and maybe wrap it in a `pytest.fixture` so that we won't create it again and again in each function.